### PR TITLE
Another slimeperson murder fix

### LINKED ISSUE #5847: holopara's healing deals tox damage to oozelings
## Round ID:

N/A

## Testmerges:
N/A

## Reproduction:
get a holopara as an oozeling give it healing abilities. watch as it heals you it also kills you with "toxins"
we all know what's going on here

### PR DESCRIPTION
**Pull request by Jeff-3**
## About The Pull Request
Closes #5847 Just adds a check to see whether or not the heal target is a jellyperson and applies the correct effect

## Why It's Good For The Game
No more accidental homicide/suicide

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

## Changelog
:cl:
fix: Holoparasites no longer kill slime people when attempting to heal
/:cl:

### CODE REVIEW

**Comment by Kapu1178** [comment]
this is what TOXLOVER is for, dont check species.

**Comment by Jeff-3** [comment]
k

File: code/modules/mob/living/simple_animal/guardian/types/support.dm [hunk 1]
```diff
@@ -32,7 +32,10 @@
 		C.adjustBruteLoss(-5)
 		C.adjustFireLoss(-5)
 		C.adjustOxyLoss(-5)
-		C.adjustToxLoss(-5, FALSE, TRUE)
+		if (!HAS_TRAIT(target, TRAIT_TOXINLOVER))
```
Comment (zeskorion) [inline code/modules/mob/living/simple_animal/guardian/types/support.dm]: Instead of this, you can just add "forced = TRUE" to that adjusttoxloss

Comment (ike709) [reply code/modules/mob/living/simple_animal/guardian/types/support.dm]: @zeskorion 
```
/mob/living/proc/adjustToxLoss(amount, updating_health = TRUE, forced = FALSE)
	if(!forced && (status_flags & GODMODE))
		return FALSE
	toxloss = CLAMP((toxloss + (amount * CONFIG_GET(number/damage_multiplier))), 0, maxHealth * 2)
	if(updating_health)
		updatehealth()
	return amount
```
How exactly is setting `forced` to `TRUE` inverting the damage for slime people?

Comment (zeskorion) [reply code/modules/mob/living/simple_animal/guardian/types/support.dm]: Setting FORCED=TRUE bypasses the toxinlover trait.

Comment (ike709) [reply code/modules/mob/living/simple_animal/guardian/types/support.dm]: Which... we don't want to do. Forced should be false.

Comment (zeskorion) [reply code/modules/mob/living/simple_animal/guardian/types/support.dm]: Ugh, im not home and cant pull it up right now. I'll go find the base code when i am

**Review by ike709 — APPROVED** [review]

**Comment by zeskorion** [comment]
@ike709 dummy, it's dealing negative damage. If forced is false, it will deal positive damage to slimepeople

File: code/modules/mob/living/simple_animal/guardian/types/support.dm (hunk 1 above)
Comment (zeskorion) [reply code/modules/mob/living/simple_animal/guardian/types/support.dm]: The intent of this pr is to make holoparas not kill slimes, right?

Comment (ike709) [reply code/modules/mob/living/simple_animal/guardian/types/support.dm]:
```
/mob/living/carbon/adjustToxLoss(amount, updating_health = TRUE, forced = FALSE)
	if(!forced && HAS_TRAIT(src, TRAIT_TOXINLOVER)) //damage becomes healing and healing becomes damage
		amount = -amount
		if(amount > 0)
			blood_volume -= 5*amount
		else
			blood_volume -= amount
	if(HAS_TRAIT(src, TRAIT_TOXIMMUNE)) //Prevents toxin damage, but not healing
		amount = min(amount, 0)
	return ..()
```
It should be false.

Comment (Jeff-3) [reply code/modules/mob/living/simple_animal/guardian/types/support.dm]: k

**Review by Archanial — APPROVED** [review]

**Pull request merged**